src/main/java/uk/gov/hmcts/darts/event/service/impl/DarNotifyServiceImpl.java

```diff
@@ -32,14 +41,27 @@ public void notifyDarPc(DarNotifyApplicationEvent event) {
 
         List<String> openCaseNumbers = caseRepository.findOpenCaseNumbers(dartsEvent.getCourthouse(), dartsEvent.getCaseNumbers());
         if (!openCaseNumbers.isEmpty()) {
+            Optional<NodeRegisterEntity> courtroomOpt = nodeRegisterRepository.findDarPcByCourtroomId(event.getCourtRoomId());
+            if (courtroomOpt.isEmpty()) {
+                log.error("Unregistered Room: message_id={}, event_id={}, courthouse={}, courtroom={}, event_timestamp={}",
```

Comment:
```suggestion
                log.error("Unable to send DAR notification. Unregistered Room: message_id={}, event_id={}, courthouse={}, courtroom={}, event_timestamp={}",
```

Reply:
https://tools.hmcts.net/jira/browse/DMP-1993 has specifically requested this error message, so i cannot change it.